### PR TITLE
Rln/update ghost anim

### PR DESCRIPTION
**Pull request by robin-norwood**
Updates to waka waka ghost sprite, could use some help making this actually good.

Also code cleanups and other fixes.

### CODE REVIEW

File: scenes/main-menu/menu.tscn [hunk 1]
```diff
@@ -6,16 +6,22 @@
...
-position = Vector2(50, 100)
 
 [node name="Exit" type="Button" parent="."]
+offset_left = 50.0
+offset_top = 800.0
+offset_right = 50.0
+offset_bottom = 800.0
 text = "Exit"
-min_size = Vector2(100, 50)
-position = Vector2(50, 800)
```
Comment (robin-norwood) [inline scenes/main-menu/menu.tscn:21]: So every time I open the project, Godot changes these to offsets. Not sure how else to deal with it other than let Godot have its way?

File: scenes/waka-waka/Scenes/maze.gd [hunk 2]
```diff
@@ -1,43 +1,50 @@
...
+    NORTH: 0, # No rotation, default orientation
+    EAST: TileSetAtlasSource.TRANSFORM_TRANSPOSE | TileSetAtlasSource.TRANSFORM_FLIP_H, # 90 degrees clockwise
+    SOUTH: TileSetAtlasSource.TRANSFORM_FLIP_H | TileSetAtlasSource.TRANSFORM_FLIP_V,   # 180 degrees clockwise
+    WEST: TileSetAtlasSource.TRANSFORM_FLIP_V | TileSetAtlasSource.TRANSFORM_TRANSPOSE  # 270 degrees clockwise
 }
 
-#FIXME: Rename "id" key as "atlas_id".
-
-# Lookup table: maps hashed PackedByteArray (NESW) to (atlas_id, # of clockwise rotations)
+## Lookup table: maps wall locations (NESW) to (atlas_id, # of clockwise rotations) for appropriate piece
```
Comment (Space-Odin) [inline scenes/waka-waka/Scenes/maze.gd:29]: Would love to talk about this on Friday just so I understand what's happening here better

File: scenes/waka-waka/Scenes/player.gd [hunk 3]
```diff
@@ -1,41 +1,48 @@
...
-	motion_mode = MOTION_MODE_FLOATING
+    velocity = Vector2.ZERO
+    motion_mode = MOTION_MODE_FLOATING
+
+    ## Shader needs some work, but kinda works like this:
+    #var shader_material = ShaderMaterial.new()
+    #shader_material.shader = load("res://scenes/waka-waka/Assets/ghost_shader.gdshader")
+    #sprite.material = shader_material
+
+    sprite.play("idle")
```
Comment (Space-Odin) [inline scenes/waka-waka/Scenes/player.gd:19]: The character's animation is only playing on movement. My read is that it should play the idle animation when not moving

**Review by Space-Odin — APPROVED** [review]

File: scenes/waka-waka/Scenes/player.gd (hunk 3 above)
Comment (robin-norwood) [reply scenes/waka-waka/Scenes/player.gd:19]: Yeah, I originally had shitty made-by AI ghost animations, and replaced them with this. Ran out of steam/decided to move on after making just the idle animation. :)

**Pull request merged**